cloud/digital_ocean/digital_ocean.py

```diff
@@ -239,10 +245,11 @@ def setup(cls, api_token):
         cls.manager = DoManager(None, api_token, api_version=2)
 
     @classmethod
-    def add(cls, name, size_id, image_id, region_id, ssh_key_ids=None, virtio=True, private_networking=False, backups_enabled=False, user_data=None):
+    def add(cls, name, size_id, image_id, region_id, ssh_key_ids=None, virtio=True, private_networking=False, backups_enabled=False, user_data=None, ipv6=False):
         private_networking_lower = str(private_networking).lower()
         backups_enabled_lower = str(backups_enabled).lower()
-        json = cls.manager.new_droplet(name, size_id, image_id, region_id, ssh_key_ids, virtio, private_networking_lower, backups_enabled_lower,user_data)
+        ipv6_lower = str(ipv6).lower()
```

Comment:
Ah, good catch. I just followed the conventions for the other values, like `private_networking` or `backups_enabled`. 

Anyways, I wasn't aware of your PR - I'll close mine and point towards yours. Also, the general unresponsiveness of the team led me away from ansible since opening my PR - IPv6 is crucial to me.

Good luck with getting your PR merged.